lib/throttle.js

```diff
@@ -25,6 +25,21 @@ module.exports = (api) => {
     };
   };
 
+  api.metasync.debounce = (
+    timeout,
+    fn,
+    args
+  ) => {
+    let timer = null;
+    return () => {
+      clearTimeout(timer);
+      timer = setTimeout(
+        () => (args !== undefined ? fn(...args) : fn()),
```

Comment:
I'd prefer line 31 to be `args = []` so that this line is just `() => fn(...args)`. That'll just make the code more readable and won't change neither contract nor functionality. An alternative is to follow @DzyubSpirit's approach (https://github.com/metarhia/MetaSync/pull/107#discussion_r115316788).